integration_tests/e2e/refer/programmeHistory.cy.ts

```diff
@@ -74,6 +74,7 @@ context('Programme history', () => {
       programmeHistoryPage.shouldHavePersonDetails(person)
       programmeHistoryPage.shouldContainNavigation(path)
       programmeHistoryPage.shouldContainBackLink(referPaths.show({ referralId: referral.id }))
+      programmeHistoryPage.shouldNotContainSuccessMessage()
```

Comment:
This should probably be added in the next test too: `shows the page without an existing programme history`